mne/datasets/_infant/base.py

```diff
@@ -60,12 +61,12 @@ def fetch_infant_template(age, subjects_dir=None, verbose=None):
     """
     # Code used to create the lists:
     #
-    # $ for name in 2-0Weeks 1-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years; do wget https://github.com/christian-oreilly/infant_template_paper/releases/download/v0.1-alpha/ANTS${name}3T.zip; done  # noqa: E501
+    # $ for name in 2-0Weeks 1-0Months 2-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years; do wget https://github.com/christian-oreilly/infant_template_paper/releases/download/v0.1-alpha/ANTS${name}3T.zip; done  # noqa: E501
     # $ md5sum ANTS*.zip
     # $ python
     # >>> import os.path as op
     # >>> import zipfile
-    # >>> names = [f'ANTS{name}3T' for name in '2-0Weeks 1-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years'.split()]  # noqa: E501
+    # >>> names = [f'ANTS{name}3T' for name in '2-0Weeks 1-0Months 2-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years'.split()]  # noqa: E501
```

Comment:
You need to actually execute this code for at least the `2-0Months` case and add the resulting file to the repo, should live in `mne/datasets/_infant/ANTS2-0Months2T.txt`

Reply:
Done.